policy/SuprimecamMapper.paf

```diff
@@ -527,17 +527,13 @@ datasets: {
         tables:        raw
         tables:        raw_visit
     }
+    # dataset defined in obs_base modified in this mapper as follows:
     forced_src_schema: {
         template:      "schema/forced_src.fits"
```

Comment:
Again, I think we should use what's in obs_base unless there's a good reason not to.